services/ssp/src/index.js

```diff
@@ -51,6 +51,47 @@ const {
 
 const app = express();
 
+const AD_RENDER_URL_START =
+  'https://privacy-sandbox-demos-dsp.dev/ads?advertiser=privacy-sandbox-demos-shop.dev&id=';
```

Comment:
NTH: how about using parameter for the root domain?

Reply:
Good catch, thanks.